cognee/infrastructure/llm/ollama/adapter.py

```diff
@@ -0,0 +1,108 @@
+import asyncio
+
+import aiohttp
+from typing import List, Type
+from pydantic import BaseModel
+import instructor
+from tenacity import retry, stop_after_attempt
+from openai import AsyncOpenAI
+
+import openai
+from cognee.infrastructure.llm.llm_interface import LLMInterface
+from cognee.infrastructure.llm.prompts import read_query_prompt
+
+
+class OllamaAPIAdapter(LLMInterface):
+    """Adapter for Ollama's API"""
+
+    def __init__(self, ollama_endpoint, api_key: str, model: str):
+
+        self.aclient =  instructor.patch(
+                                                AsyncOpenAI(
```

Comment:
Why do we need OpenAI here?

Reply:
yup, uses openai client as default client, they all do things in their format spec